### PR TITLE
Fixing categories typo in methodologies

### PR DESCRIPTION
**Pull request by molfinn**
### Added

| Page | Deploy Preview | Comment |
| ----- | ----- | ----- |
| Name | Link | Comment |

### Updated

| Page | Deploy Preview | Comment |
| ----- | ----- | ----- |
| Name | Link | Comment |

## Preview This Change

To see how this change looks in production, scroll down to **Deploy Preview**. Select the link that looks like `https://deploy-preview-<num>--cobalt-docs.netlify.app/`

## Variables

Help us support a “Write once, publish everywhere” single source of truth. If you see a line that looks like:

`{{% asset-categories %}}`

You’ve found a [shortcode](https://gohugo.io/content-management/shortcodes/) that we include in multiple documents.

You’ll find the content of the shortcode in the following directory:

https://github.com/cobalthq/cobalt-product-public-docs/tree/main/layouts/shortcodes

That shortcode has the same base name as what you see in the PR, such as `asset-categories.html`.

## Checklist for PR Author

[ ] Did you check for broken links and alt text?

Be sure to check for broken links and Alt Text issues. We have a partially automated process,
as described in this section of our repository README:
[Test Links and Alt Attributes](https://github.com/cobalthq/cobalt-product-public-docs/blob/main/README.md#test-links-and-alt-attributes).

### CODE REVIEW

File: content/en/Methodologies/_index.md [hunk 1]
```diff
@@ -30,6 +30,7 @@ asset.
 - [External Network](/methodologies/external-network/)
 - [Cloud](/methodologies/cloud/)
 - [Azure AD](/methodologies/azure-ad/)
+- [Desktop] (/methodologies/desktop/)
```
Comment (reginandrade) [inline content/en/Methodologies/_index.md:33]: I think the space btwn ] and ( is causing this not to be rendered as a link, but rather plain text

**Review by reginandrade — CHANGES_REQUESTED** [review]
[Desktop] (/methodologies/desktop/)  i think should be:
[Desktop](/methodologies/desktop/)

**Comment by molfinn** [comment]
@reginandrade i think I updated based on your feedback! I also had one more commit - to add the params to the Planning page.

**Comment by molfinn** [comment]
Closing this pr. Work was merged in 458.

**Pull request closed without merging**